config/software/ruby-cleanup.rb

```diff
@@ -69,7 +69,7 @@
 
   block "Remove leftovers from compiling gems" do
     # find the embedded ruby gems dir and clean it up for globbing
-    target_dir = "#{install_dir}/embedded/lib/ruby/gems/*/".tr('\\', "/")
+    target_dir = "#{install_dir}/embedded/lib/ruby/gems/*/".tr("\\", "/")
```

Comment:
Where did this change come from? I feel like `\\` has very special behavior when used in double quotes. Is this fixing an error of some kind?

Reply:
it is fixing the verify build pipeline failure https://buildkite.com/chef-oss/chef-omnibus-software-master-verify/builds/744